### PR TITLE
Sparkpost

### PR DESCRIPTION
**Pull request by rajumsys**
For internal review, once completed, we'll open the PR against the official repo. 

The following app was deployed using this tutorial http://hello-sparkpost.azurewebsites.net/

### CODE REVIEW

File: articles/app-service-web/nodejs-send-email-with-sparkpost.md [hunk 1]
```diff
@@ -0,0 +1,412 @@
...
+
+## Prerequisites
+- A Microsoft Azure Account. If you don't have an account, you can sign up for a free trial
+- A SparkPost Account
+- Node.js 4x
+- Git
+- Azure CLI
+
+
+## What is the SparkPost?
```
Comment (yargevad) [inline articles/app-service-web/nodejs-send-email-with-sparkpost.md]: -the => "What is SparkPost?"

File: articles/app-service-web/nodejs-send-email-with-sparkpost.md [hunk 2]
```diff
@@ -0,0 +1,412 @@
...
+SparkPost is the world’s fastest-growing email delivery service, providing a developer friendly robust cloud API for apps and websites to send and receive email.
+
+
+For more information about SparkPost, visit [https://sparkpost.com](https://sparkpost.com).
+
+## Create a SparkPost Account
+SparkPost provides 100K free emails per month including access to APIs and analytics. This free account is enough to get started. However, feel free to choose [appropriate package][sparkpost packages] based on your sending volume requirements.
```
Comment (yargevad) [inline articles/app-service-web/nodejs-send-email-with-sparkpost.md]: "choose **the** appropriate package"

Comment (yargevad) [reply articles/app-service-web/nodejs-send-email-with-sparkpost.md]: If there aren't any specific requirements around how this is worded, I'd prefer simpler language: "your sending volume requirements" => "how many emails you'll be sending"

File: articles/app-service-web/nodejs-send-email-with-sparkpost.md [hunk 3]
```diff
@@ -0,0 +1,412 @@
...
+
+
+### To sign up for a SendGrid account
```
Comment (yargevad) [inline articles/app-service-web/nodejs-send-email-with-sparkpost.md]: SparkPost!

File: articles/app-service-web/nodejs-send-email-with-sparkpost.md [hunk 4]
```diff
@@ -0,0 +1,412 @@
...
+```
+
+![run-locally][run-locally]
+
+
+> [!TIP]
+> To run the application on different port, run the command with desired port like
+>
+> ```bash
+> PORT=3000 SPARKPOST_API_KEY=<API_KEY> npm start
```
Comment (yargevad) [inline articles/app-service-web/nodejs-send-email-with-sparkpost.md]: Missing closing triple backticks?

File: articles/app-service-web/nodejs-send-email-with-sparkpost.md [hunk 5]
```diff
@@ -0,0 +1,412 @@
...
+
+-----------------------
+
+## Log in to Azure
+
+We are now going to use the [Azure CLI](azure-cli) in terminal to create the resources needed to deploy our above sample Node.js application to Azure. Log in to your Azure subscription with the [az login](/cli/azure/#login) command and follow the on-screen directions.
```
Comment (yargevad) [inline articles/app-service-web/nodejs-send-email-with-sparkpost.md]: "in a terminal"

File: articles/app-service-web/nodejs-send-email-with-sparkpost.md [hunk 6]
```diff
@@ -0,0 +1,412 @@
...
+
+You should be able to browse your newly created web application using the following URL.
+
+```
+http://<app_name>.azurewebsites.net
+```
+
+![default-app][default-app]
+
+This is default web app in Azure. We'll now configure it to use Node.js and then push our sample app.
```
Comment (yargevad) [inline articles/app-service-web/nodejs-send-email-with-sparkpost.md]: "This is **the** default web app"

File: articles/app-service-web/nodejs-send-email-with-sparkpost.md [hunk 7]
```diff
@@ -0,0 +1,412 @@
...
+
+## Set SparkPost API Key in environment variable
+In an earlier step, we've created API Key. Now we'll save this API Key to application's settings which will be available to our codes via environment variables. In our code we've already referenced to this environment variable, hence, it should work without any further modification in code.
```
Comment (yargevad) [inline articles/app-service-web/nodejs-send-email-with-sparkpost.md]: "we've created **an** API Key"
"already referenced this environment variable" (remove **to**)

File: articles/app-service-web/nodejs-send-email-with-sparkpost.md [hunk 8]
```diff
@@ -0,0 +1,412 @@
...
+
+```azurecli
+az appservice web config appsettings update --name <app_name> --resource-group testResource --settings SPARKPOST_API_KEY=<api_key>
+```
+
+>[!Tip]
+> If you ever modify API Key, you'll only need to update this environment variable. No code update is necessary.
```
Comment (yargevad) [inline articles/app-service-web/nodejs-send-email-with-sparkpost.md]: "modify **your** API Key"

File: articles/app-service-web/nodejs-send-email-with-sparkpost.md [hunk 9]
```diff
@@ -0,0 +1,412 @@
...
+az appservice web config update --linux-fx-version "NODE|4.4.7"  --name <app_name> --resource-group testResource
+```
+
+## Configure local git deployment
+
+```azurecli
+az appservice web source-control config-local-git --name <app_name> --resource-group testResource --query url --output tsv
+```
+
+It'll return an git repo URL. Copy it as we'll use this in next step.
```
Comment (yargevad) [inline articles/app-service-web/nodejs-send-email-with-sparkpost.md]: "return an git" => "return **a** git"
"use this in **the** next step"

File: articles/app-service-web/nodejs-send-email-with-sparkpost.md [hunk 10]
```diff
@@ -0,0 +1,412 @@
...
+## Push to Azure from git
+Now that we've added git remote, we can simply push codes to this origin.
+
+```bash
+git push azure master
+```
+
+Once prompted, enter the password we've used during creation of deployment user.
+
+During deployment, Azure App Service will communicate it's progress with Git.
```
Comment (yargevad) [inline articles/app-service-web/nodejs-send-email-with-sparkpost.md]: "it's" => "**its**"

**Review by yargevad — COMMENTED** [review]
first pass

File: articles/app-service-web/nodejs-send-email-with-sparkpost.md (hunk 3 above)
Comment (rajumsys) [reply articles/app-service-web/nodejs-send-email-with-sparkpost.md]: Yikes!

**Comment by rajumsys** [comment]
Closing it and going to create new PR against original repo.

**Pull request closed without merging**